corehq/apps/linked_domain/remote_accessors.py

```diff
@@ -65,8 +66,10 @@ def _format_missing_media(media):
 
         still_missing_media = _get_missing_multimedia(app)
         if still_missing_media:
-            return False
-    return True
+            raise MultimediaMissingError(_(
+                'Application has missing multimedia even after an attempt to pull them. '
+                'An email has been sent with details. Please try again. If persists, report an issue.'
```

Comment:
we removed the email now? so this text needs to change?

Reply:
I thought this meant we send an email to the user (though I never looked for it...). If this is referencing the soft assert email, I'll update the message.

Reply:
340382cc21d